scripts/Deepcell_Postprocessing.ipynb

```diff
@@ -13,17 +13,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import numpy as np\n",
     "import os\n",
-    "import skimage.io as io\n",
-    "import matplotlib.pyplot as plt\n",
     "import sys\n",
-    "import pandas as pd\n",
     "\n",
-    "sys.path.append(\"../\")\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import skimage.io as io\n",
+    "import xarray as xr\n",
     "\n",
-    "from segmentation.utils import data_utils, segmentation_utils\n",
-    "import xarray as xr"
+    "from segmentation.utils import data_utils, segmentation_utils"
```

Comment:
Any particular reason to change the order of these imports, just code cleanup I assume?

Reply:
Yup, just alphabetizing and separating based on import type